cads_adaptors/tools/post_processors.py

```diff
@@ -60,18 +60,22 @@ def daily_reduce(
     how: str | Callable = "mean",
     **kwargs,
 ) -> dict[str, Dataset]:
-    from earthkit.aggregate import temporal
+    from earthkit.transforms.aggregate import temporal
 
     out_xarray_dict = {}
     for in_tag, in_dataset in in_xarray_dict.items():
         out_tag = f"{in_tag}_daily-{how}"
         context.add_stdout(f"Daily reduction: {how} {kwargs}")
         context.add_user_visible_log(f"Temporal reduction: {how} {kwargs}")
-        out_xarray_dict[out_tag] = temporal.daily_reduce(
+        reduced_data = temporal.daily_reduce(
             in_dataset,
             how=how,
             **kwargs,
         )
+        out_xarray_dict[out_tag] = update_history(
+            reduced_data,
+            f"earthkit.transforms.aggregate.temporal.daily_reduce({in_tag}, how={how} **{kwargs})",
```

Comment:
Should there be a comma here?
```suggestion
            f"earthkit.transforms.aggregate.temporal.daily_reduce({in_tag}, how={how}, **{kwargs})",
```